File: 1822-sign-of-the-product-of-an-array/1822-sign-of-the-product-of-an-array.py
class Solution:
    def arraySign(self, nums: List[int]) -> int:
        neg = 0
        for n in nums:
            if n == 0:
                return 0
            elif n < 0:
                neg += 1
        
        return 1 if neg % 2 == 0 else -1

File: 1822-sign-of-the-product-of-an-array/1822-sign-of-the-product-of-an-array.c
int arraySign(int* nums, int numsSize){
    int res = 1;
    for (int i = 0; i < numsSize; i++) {
        if (nums[i] == 0)
            return 0;
        else if (nums[i] < 0) 
            res *= -1;
    }
    return res;
}